README.md

```diff
@@ -1,58 +1,70 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# TheMealDB API Capstone
+This project is created using HTML5,SCSS and JavaScript with TheMealDB API that accesses the meals from the api and displays it. some of the features are: user can see the meals and add and see his/her comment.The user can also reserve the meals. 
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- SCSS
+- Modern JavaScript
+- Linters
+- Webpack bundler
+- Kanban Board
+- TheMealDB API
 
-## Live Demo (if available)
+![](https://github.com/yasinabdmahmood/JavaScript-Group-Capstone/blob/dev/screen.png)
+## Live Demo 
+[Live Demo Link](https://yasinabdmahmood.github.io/JavaScript-Group-Capstone/dist/)
 
-[Live Demo Link](https://livedemo.com)
+## Discription video
+ [Demonstration Video](https://drive.google.com/file/d/1Oxs7U9y3OCxFtat40MboR6CnP1MrhFR_/view?usp=sharing)
```

Comment:
- [ ] Great job on the overall project team, but the video description requires some changes. Kindly ensure that all 3 members that created the project are present and participate in the video
- [ ] Please also make sure you mention and show a specific piece of code you're proud of in addition to the overall app.